tests/remote/remote_controller.pm

```diff
@@ -19,6 +19,8 @@ sub run() {
     my $target_ip;
 
     select_console 'root-console';
+    type_string "chown $username /dev/$serialdev\n";
```

Comment:
my guess is that assert_script_run won`t work here, as it uses $serialdev where it has no access (chicken&egg) - but I will check :)